reference/kabusapi/REST_API.md

```
@@ -51,6 +70,80 @@
+| ClosePositionOrder | int |  | 建玉整理順序 |
+| ClosePositions | array |  | 個別建玉指定 ({ HoldID, Qty }) |
+| FrontOrderType | int | ✓ | 10:成行, 20:指値 … |
+| Price | int | ✓ | 注文価格 (成行時は0) |
+| ExpireDay | int | ✓ | 有効期限 (YYYYMMDD, 当日0) |
+| ReverseLimitOrder | object |  | 逆指値 ({ TriggerSec, TriggerPrice, UnderOver, AfterHitOrderType, AfterHitPrice }) |
+
+成功レスポンス `OrderSuccess` → `{ "OrderId": "…", "Result": 0 }`
+
+> ※発注系エンドポイントは秒間 5 req 程度のレート制限があります。
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)